lib/models/item_menu_action.dart

```diff
@@ -96,6 +100,11 @@ extension ItemMenuActionExtension on ItemMenuAction {
         return AppLocalizations.of(context).copy;
       case ItemMenuAction.share:
         return AppLocalizations.of(context).share;
+      case ItemMenuAction.toggleVisited:
+        final bool visited = ref.read(visitedProvider(id)).value ?? false;
+        return visited
+            ? AppLocalizations.of(context).markUnRead
```

Comment:
Please rename to markUnread (note the lowercase letter).

Reply:
done!